### PR TITLE
fix: application template

### PR DESCRIPTION
**Pull request by Jakub-Vacek**
## Description
This PR fixes problem with incorrect location of application code and updates application code template to be aligned with OneSdk examples

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have updated the documentation accordingly. For updating Oclif commands documentation use [oclif-dev](https://github.com/oclif/dev-cli#oclif-dev-readme).
- [ ] I have read the **CONTRIBUTION_GUIDE** document.
- [ ] I haven't repeated the code. (DRY)
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: src/logic/application-code/js/application-code.ts [hunk 1]
```diff
@@ -38,33 +38,45 @@ export function jsApplicationCode(
...
+    try {
+      // Execute use case
+      const result = await useCase.perform(
+        // Use case input
+        ${input},
+        {
+          provider: '${provider}',
+          parameters: ${parametersString},
+          // Security values for provider
+          security: ${securityString}
```
Comment (kysely) [inline src/logic/application-code/js/application-code.ts:70]: If there are no parameters or no security, shouldn't these be omitted altogether?

File: src/commands/map.ts [hunk 2]
```diff
@@ -117,8 +117,16 @@ export default class Map extends Command {
       saved ? 'Boilerplate code prepared.' : 'Boilerplate code already exists.'
     );
 
+    ux.start('Setting up local project');
     // TODO: install dependencies
     await prepareJsProject(undefined, undefined, { logger });
+
+    ux.succeed(
+      // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+      `Local project set up. You can now install defined dependencies and run \`superface execute ${
```
Comment (kysely) [inline src/commands/map.ts:126]: question: Will we also provider instructions for installing dependencies per target language?

**Review by kysely — APPROVED** [review]
Looks okay 👍

File: src/logic/application-code/js/application-code.ts (hunk 1 above)
Comment (Jakub-Vacek) [reply src/logic/application-code/js/application-code.ts:70]: We can do that. I wanted to illustrate how is security passed to OneClient call 🤷🏻‍♂️

**Pull request merged**